code/modules/cargo/packs.dm

```diff
@@ -303,6 +303,28 @@
 					/obj/item/clothing/suit/armor/vest)
 	crate_name = "armor crate"
 
+/datum/supply_pack/security/assorted_lens
+	name = "Assorted external lens Crate"
+	desc = "Contains five experimental lens that can be applied to standard laser guns. Requires Security access to open."
+	cost = 8000
+	contains = list(/obj/item/external_lens/BTC,
```

Comment:
Don't use uppercase in paths, also give these thing names that actually mean something.